renovate-configs/dee-platform-ops.json5

```diff
@@ -0,0 +1,86 @@
+// Shared Renovate Base Configuration for repos operated by the
+// @GoogleCloudPlatform/dee-platform-ops team.
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:recommended",
+  ],
+  // Rate limits, to protect maintainers (and CI) from PR floods.
+  "prHourlyLimit": 10,
+  "prConcurrentLimit": 30,
```

Comment:
**Thought:** I'm guessing this `"prConcurrentLimit": 30,` is _per repo_. That is, at any given time, we can have 30 Renovate PRs in Bank of Anthos, 30 in Online Boutique, 30 in GKE samples, etc.
I'd lower this number to about 10 — since grouping by language also reduces the total repo count.

Reply:
Yes, your understanding is correct, this is a per-repo number.
I admit this number is pretty arbitrary, and I appreciate your justification.
In regular practice, we should come nowhere near this number, because of the grouping, as you noted.

As I see it, the consequence for exceeding this number is a potentially long delay before the update PRs are created, since we're only running once each week. However, there are 2 reasons I think its ok to reduce this somewhat:
1) the dependency dashboard issue will still show rate-limited updates, and will create them when requested.
2) This is a base configuration, and each repo can override it.

What do you think about 15 as a starting point? we may want to tune it lower for OB.

Reply:
15 sounds good!